web/Cargo.toml

```diff
@@ -74,7 +74,7 @@ features = [
     "EventTarget", "GainNode", "Headers", "HtmlCanvasElement", "HtmlDocument", "HtmlElement", "HtmlFormElement",
     "HtmlInputElement", "HtmlTextAreaElement", "KeyboardEvent", "Location", "PointerEvent",
     "Request", "RequestInit", "Response", "Storage", "WheelEvent", "Window", "ReadableStream", "RequestCredentials",
-    "Url", "Clipboard",
+    "Url", "Clipboard", "FocusEvent"
```

Comment:
Not directly related to this PR but there seems to have once been a clear attempt to alphabetize this but once ReadableStream was added features were just added in the order they became needed and that has since continued.

Reply:
The biggest problem of keeping the list in a lexicographical order with multiple entries per line is that most of the time adding/removing a single feature will cause multiple lines to be edited and rearranged